backend/src/routes/units.ts

```diff
@@ -0,0 +1,17 @@
+import express from "express";
+const router = express.Router();
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function, @typescript-eslint/no-unused-vars
+router.get("/units/:id", (req, res) => {});
```

Comment:
Is the Get Unit Route coming in another PR?

Reply:
Just to add on to this. Not sure if we want to have the commented section below merged into main

Reply:
Added the get Unit route.